defaults/main.yml

```diff
@@ -11,6 +11,12 @@
 gitlab_dependencies: True
 
 
+# .. envvar:: gitlab_use_systemd
+#
+# Should the ``debops.gitlab`` role install systemd service files instead of the upstream init script
+gitlab_use_systemd: False
```

Comment:
I'm going to test the PR on `systemd`, but I think if it works fine, you could enable the support on `systemd`-hosts right away, with something like:
```yaml
gitlab_use_systemd: '{{ True if ansible_service_mgr == "systemd" else False }}'
```
Although, does the role support switching from one to the other? I wonder if it should, it might be complicated. I would maybe go with an Ansible facts based approach and first save the current init type in the `gitlab.fact` file, and use that to set the default.